src/pyodide/python-entrypoint-helper.js

```diff
@@ -76,7 +73,11 @@ async function setupPackages(pyodide) {
       .values()
       .map(({ packageData }) => packageData);
   }
-  const transitiveRequirements = new Set(packageDatas.map(({ name }) => name));
+  return new Set(packageDatas.map(({ name }) => name));
```

Comment:
We should probably grab the `normalizedName` here so we don't have to renormalize it ourselves:
```suggestion
  return new Set(packageDatas.map(({ normalizedName }) => normalizedName));
```

Reply:
Hm, that fails on edgeworker since normalizedName doesn't seem to be defined there. Imma just leave it the way it is if that's ok

Reply:
We can clean up later is fine.